src/loop_activity.cpp

```diff
@@ -0,0 +1,40 @@
+#include "ybadge.h"
+
+void loop_exploration();
+void loop_challenge1();
+void loop_challenge2();
+void loop_challenge3();
+
+void loop_activity() {
+    // This function should call ONE of your functions below.
+    // Uncomment or comment the lines so that just the function
+    // you want to call is uncommented.
+
+    loop_exploration();
+    // loop_challenge1();
+    // loop_challenge2();
+    // loop_challenge3();
+}
+
+void loop_exploration() {
+    // Add your code here for exploration activities
+    while(true) {
+        for(int i = 20; i >= 1; --i) {
```

Comment:
Is this code supposed to be here?

Reply:
No. I'll get it fixed after the block of meetings. Thanks for the catch. I was taking pics for the documentation and forgot to delete the src code.

Reply:
Done